app/controllers/publishers/vacancies_controller.rb

```diff
@@ -59,8 +61,25 @@ def summary
     @feedback_form = Publishers::JobListing::FeedbackForm.new
   end
 
+  def job_applications
+    @vacancy = VacancyPresenter.new(@vacancy)
+    @application_statuses = vacancy_applications.group_by { |k| k[:status] }.transform_values(&:count)
+  end
+
   private
 
+  def vacancy_applications
+    @vacancy_applications ||= @vacancy.job_applications.order("#{sort.column} #{sort.order}").reject { |a| a[:status] == "draft" }
```

Comment:
```ruby
vacancy.job_applications.not_draft.order(sort.column => sort.order)
```
We can scope by job application status, and we only need to do an interpolated order when we want to sort by something on a joined table (e.g `vacancies.publish_on`)